### ISSUE TITLE
update for several resources

### ISSUE THREAD

**Issue by Stryzhevskyi** [sebasgarcep/firebase-admin-mock#9]
Hi,
It seems update works wrong: in this example I expected both `b` and `c` should have new value.

``` javascript
const {AdminRoot} = require('firebase-admin-mock');
const admin = new AdminRoot();
admin.initializeApp({
   databaseUrl: 'test-url',
});
const db = admin.database();
db.setMockData({
   a: {
       b: 'x',
       c: 'y'
   }
});
return db.ref('/a')
   .update({
       b: '-',
       c: '-',
   })
   .then(() => db.ref('/a').once('value'))
   .then(snapshot => {
       const val = snapshot.val();

       expect(val.c).toEqual('-');
       expect(val.b).toEqual('-');
   });
```

In my environment only `c` has been updated.
Version: 0.0.8

Thank you for great project!

**Comment by sebasgarcep** [comment]
@Stryzhevskyi I'm currently tied up. Would you mind sending a pull request?

**Comment by sebasgarcep** [comment]
@Stryzhevskyi We've already solved this in `v0.0.9`.  😄
